docs/site/content/docs/guides/microfrontends.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Microfrontends
+description: Learn how to develop microfrontends with Turborepo.
+---
+
+import { PackageManagerTabs, Tab } from '#components/tabs';
+import { Callout } from '#components/callout';
+import { Files, File, Folder } from '#components/files';
+import { ExperimentalBadge } from '#components/experimental-badge';
+
+<ExperimentalBadge>Experimental</ExperimentalBadge>
+
+Turborepo simplifies local development for vertical microfrontends by coordinating multiple applications and providing a unified development experience.
+
+Vertical microfrontends are an architectural pattern where a web application is composed of multiple frontend applications. Each microfrontend can be developed, tested, and deployed separately, while appearing as a single cohesive application to your users.
+
+## How Turborepo helps
+
+Turborepo enhances microfrontend development with:
+
+- **Unified development experience**: Start all your microfrontends with a single command
```

Comment:
I feel like the unified devx we offer is that you can browse all of your microfrontends from a single host locally. Maybe we link to https://vercel.com/docs/microfrontends/local-development#the-need-for-a-microfrontends-proxy explaining why this is helpful?